### DISCUSSION TITLE
so-rule-update with errors

### DISCUSSION THREAD

**Discussion by bit1290** [Security-Onion-Solutions/securityonion#6875] [Unsupported Versions] [answered]
Hello,

I'm having the following error when trying to run an so-rule-update:

![image](https://user-images.githubusercontent.com/16745902/149558297-aea5ea66-c016-45ea-9dfb-cfeae3fcb7b3.png)

This is from a rule that comes from Emerging Threats.
Is there some way to fix this kind of exceptions?

Thanks :)

**Comment by dougburks** [comment, marked as answer]
Are you sure that rule is from Emerging Threats? Emerging Threats rules usually start with `ET` instead of `BDS`. If you've modified your rule config, please revert your changes and try again.
